Wiki/text-notes.md

```diff
@@ -1,107 +1,94 @@
+1. Selecting text to bring up an inline toolbar.
 
+%%{WARNING}%% Image does not exist in repo
 ![](api/images/aQT4C1G1rjUk/text-notes-formatting-block.pn)
 
-Read only vs. editing mode
---------------------------
+2. Clicking on the block toolbar.
 
-Text notes are normally opened in edit mode, however there are two cases when they are open in read-only mode:
 
-*   they are long and thus would take time to load so by default we open them as read only which is much quicker
-*   or the note has `readOnly` [label](attributes.md)
+## Read-Only vs. Editing Mode
 
-In both cases, it is possible to switch to editable mode again.
+Text notes are usually opened in edit mode. However, they may open in read-only mode under the following circumstances:
 
-General Formatting
-------------------
+- The note is long and would take time to load, so it is opened in read-only mode by default for quicker access.
+- The note has a `readOnly` [label](attributes.md).
 
-Trilium uses the CKEditor, so any formatting that the CKEditor supports should be available in Trilium. For example:
+In both cases, it is possible to switch back to editable mode.
 
-**Bold** – Type `**text**` or `__text__`
+## General Formatting
 
-_Italic_ – Type `*text*` or `_text_`
+Since Trilium uses CKEditor, any formatting supported by CKEditor is available. Examples include:
 
-`Code` – Type \`text\`
-
-~Strikethrough~ – Type `~~text~~`
+- **Bold**: Type `**text**` or `__text__`
+- _Italic_: Type `*text*` or `_text_`
+- `Code`: Type \`text\`
+- ~Strikethrough~: Type `~~text~~`
 
 ### Lists
 
-*   Bulleted list – Start a line with `*` or `-` followed by a space
-
-1.  Numbered list – Start a line with `1.` or `1)` followed by a space
-
-\[ \] To-do list – Start a line with `[ ]` or `[x]` followed by a space to insert an unchecked or checked list item, respectively
+- Bulleted list: Start a line with `*` or `-` followed by a space
+- Numbered list: Start a line with `1.` or `1)` followed by a space
+- To-do list: Start a line with `[ ]` for an unchecked item or `[x]` for a checked item
 
 ### Blocks
 
-> Block quote – Start a line with `>` followed by a space
-
-`Multi-line Code block` – Start a line with \`\`\`
-
-### Other
-
-Headings – Start a line with `##` or `###` followed by a space to create a heading 1, heading 2, or heading 3 (up to heading 6 if options defines more headings)
-
-Note: Trilium only accepts headings with `##` and more because `#` is reserved for the title
+- Block quote: Start a line with `>` followed by a space
 
-Horizontal line – Start a line with `---`
------------------------------------------
+### Multi-Line Code Blocks
 
-Markdown & Autoformat
----------------------
+To create a multi-line code block, start a line with "\`\`\`[lang]", for example:
 
-CKEditor supports markdown-like editing experience. It recognizes syntax and automatically converts it to rich text. See it in action:
+```js
+if (1 > 2) {
+    console.log("Error in the matrix");
+}
+```
```

Comment:
I can't preview here, but since the Docs are rendered from markdown, won't this example block show up as an actual code block rather than showing how to make a code block? Don't the backticks need to be escaped here to show how it's done?
Just copied and pasted:
```js
if (1 > 2) {
    console.log("Error in the matrix");
}
```

Reply:
The line above describes it.

> To create a multi-line code block, start a line with "\`\`\`[lang]", for example: